src/extension-installations/reducers.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers } from 'redux';
+import { storage, collection, one } from '@shoutem/redux-io';
+import { INSTALLATIONS } from '../resources/extensionInstallations';
+
+export default combineReducers({
+  storage: storage(INSTALLATIONS),
+  current: one(INSTALLATIONS, 'current'),
```

Comment:
This will create problems when multiple settings pages are running in the same context. This is not valid at the moment, but we should try to figure out how to organize the state so that it can work in this context as well.